### PR TITLE
Nginx changes in dp-config-aws chart

### PR DESCRIPTION
**Pull request by suyoggandhi-tibco**

### CODE REVIEW

File: charts/dp-config-aws/Chart.lock
Comment (syan-tibco) [inline charts/dp-config-aws/Chart.lock:1]: you need to remove this file right?

File: charts/dp-config-aws/Chart.yaml [hunk 1]
```diff
@@ -9,11 +9,15 @@ maintainers:
...
     email: cloud@tibco.com
 dependencies:
-  - name: traefik
-    version: 24.0.0
-    repository: https://traefik.github.io/charts
-    condition: traefik.enabled
+  # - name: traefik
+  #   version: 24.0.0
+  #   repository: https://traefik.github.io/charts
+  #   condition: traefik.enabled
```
Comment (syan-tibco) [inline charts/dp-config-aws/Chart.yaml:15]: keep the traefik we just disable it on chart values

File: charts/dp-config-aws/values.yaml [hunk 2]
```diff
@@ -12,11 +12,11 @@ dns:
...
 httpIngress:
   enabled: true
-  name: traefik
+  name: nginx
   ingressClassName: alb
   backend:
-    serviceName: dp-config-aws-traefik
-    servicePort: 443
+    serviceName: ingress-nginx-controller
+    servicePort: 80
```
Comment (syan-tibco) [inline charts/dp-config-aws/values.yaml:19]: add comment like 
```
  # traefik for traefik
  name: nginx 
  ingressClassName: alb
  backend:
    # dp-config-aws-traefik for traefik
    serviceName: ingress-nginx-controller 
    servicePort: 80
```

File: charts/dp-config-aws/values.yaml [hunk 3]
```diff
@@ -79,23 +79,31 @@ storageClass:
...
+#         - traefik
+#         - web
+#         - websecure
+#   providers: # for external service
+#     kubernetesIngress:
+#       allowExternalNameServices: true
+#   nodeSelector: # for cic2
+#     kubernetes.io/os: linux
+#   additionalArguments:
+#     - '--serversTransport.insecureSkipVerify=true'
```
Comment (syan-tibco) [inline charts/dp-config-aws/values.yaml:101]: you don't need to comment it out. as long as we set default as false.

**Review by syan-tibco — COMMENTED** [review]
chart version in Chart.yaml needs to be updated.

File: charts/dp-config-aws/Chart.lock
Comment (suyoggandhi-tibco) [reply charts/dp-config-aws/Chart.lock:1]: Removed file.

**Pull request merged**